aiger_analysis/safety_game.py

```diff
@@ -46,21 +46,26 @@ def inputs(self):
 
     def is_realizable(self, use_cegar=False, verbose=False):
         assert len(self.aig.outputs) is 1
-        bad = atom(False)
-        single_step = BoolExpr(_cutlatches_and_rename(self.aig) >>
-                               aiger.sink(self.aig.latches))
+        initial_state = {x: val for (x, val) in self.aig.latch2init}
```

Comment:
Can just do:
`initial_state = dict(self.aig.latch2init)`

This has the additional benefit of working even if `self.aig.latch2init` becomes a dict (or some sort of `FrozenDict`) as planned.